std/meta.d

```diff
@@ -1309,6 +1309,65 @@ private template staticMerge(alias cmp, int half, Seq...)
     }
 }
 
+/**
+ * Allows subsetting of alias sequence that are not
+ * necessarily adjacent to each other, i.e. cannot be sliced.
+ *
+ * Parameters:
```

Comment:
The standard section name is `Params`, but it looks like the rest of this file got it wrong as well.